### PR TITLE
Address #142

### PR DESCRIPTION
**Pull request by Tguntenaar**
_Resolves #142_
Address #142 and make sure not to introduce #210 and #213.

Expected behavior

1. User should see the same fields & preview in the solution reply as shown in the create solution post
2. User should be able to select ~USDC~ USDT and proceed successfully without a warning message

[Testnet Preview](https://test.near.social/thomasguntenaar.testnet/widget/gigs-board.pages.Feed)
And a  mainnet [preview](https://near.org/thomasguntenaar.near/widget/gigs-board.pages.Feed)

### CODE REVIEW

File: src/gigs-board/pages/Create.jsx [hunk 1]
```diff
@@ -89,18 +89,18 @@ const labels = labelStrings.map((s) => {
...
-  //
+
   author_id: context.accountId,
   // Should be a list of objects with field "name".
   labels,
   // Should be a list of labels as strings.
   // Both of the label structures should be modified together.
   labelStrings,
-  postType: "Idea",
+  postType: "",
```
Comment (ailisp) [inline src/gigs-board/pages/Create.jsx:99]: This should be unchanged otherwise it cause bug #213

**Comment by ailisp** [comment]
PR looks good overall. The description isn't accurate, this PR intends to address #142 without introduce bugs. The acceptance criteria is
> Expected behavior

 > User should see the same fields & preview in the solution reply as shown in the create solution post
    User should be able to select USDC and proceed successfully without a warning message

As i tested these criteria are met, so just ensure address above comments to avoid introduce a bug.

**Review by ailisp — CHANGES_REQUESTED** [review]
Please address above comment to avoid introduce #213

**Comment by ailisp** [comment]
@Tguntenaar I just update the title and acceptance criteria to make this PR's purpose accurate

**Comment by ailisp** [comment]
Reply solution post doesn't include the "yes and no" section. Also please merge main branch and resolve conflict. Now main branch has only USDT and NEAR as currency.

**Comment by Tguntenaar** [comment]
@ailisp The preview is probably showing the wrong version. Something with the bos cli blocks me from pushing the components to mainnet.

**Comment by ailisp** [comment]
There are some bugs during my testing, list here for checking after fix:
- [ ] reply with sponsorship type, specify NEAR token gives this error:
image
- [ ] reply with solution type, specify USDT token renders preview and result post with "[object object]:
image

Below are correct cases, but need ensure it's still work after fix the above two issues.
- [ ] reply with sponsorship type, specify USDT token works fine. 
- [ ] reply with solution type, specify NEAR token works fine.
- [ ] create solution post with NEAR or USDT token work fine.

**Comment by Tguntenaar** [comment]
- [x] reply with sponsorship type, specify NEAR token no longer gives error
- [x] reply with solution type, specify USDT token renders preview and result post correct

Below are correct cases, but need ensure it's still work after fix the above two issues.

- [ ] reply with sponsorship type, specify USDT token works fine.
- [ ] reply with solution type, specify NEAR token works fine.
- [ ] create solution post with NEAR or USDT token work fine.

**Comment by Tguntenaar** [comment]
Hi @ailisp,

Finally tested all cases, they all work. I think it is ready to merge.

**Comment by ailisp** [comment]
@Tguntenaar Mainnet preview no longer works, can you push a new one?

Testnet preview has some bugs:
- reply with a solution, when requested sponsor is empty, the requested fund doesn't show up:
image

- reply with a solution, enter the requested sponsor, and leave token as default (NEAR), but the transaction shows USDT:
image

**Comment by Tguntenaar** [comment]
Hi @ailisp,

> reply with a solution, when requested sponsor is empty, the requested fund doesn't show up

This was intented it is the same behaviour as the Create widget. (Create a solution post and remove the requested sponsorship field) We could change it in both places. What would be the desired behaviour? A warning before submission that sponsorship is required or did you have something else in mind. 

> reply with a solution, enter the requested sponsor, and leave token as default (NEAR), but the transaction shows USDT

That is strange.. I didn't and don't experience this. It is probably caused by the snapshot of the state. I will try to reproduce the behaviour and come up with a solution.

**Comment by ailisp** [comment]
OK! Sounds great then. I also tested on mainnet and the NEAR shown as USDT problem doesn't exist. For sponsorship empty it's not a new problem introduced in this PR, we might want to address in a separate ticket. Now all tests work fine. Could you please resolve conflict so we can merge it?

**Review by ailisp — APPROVED** [review]

**Pull request merged**